TestPlatform.sln

```diff
@@ -159,6 +159,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "perf", "perf", "{0D4DF78D-7
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "vstest.console.PlatformTests", "test\vstest.console.PlatformTests\vstest.console.PlatformTests.csproj", "{8C068694-23A2-47A2-A0DD-DB82D0AF0142}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DisableAppdomainTest1", "test\TestAssets\DisableAppdomainTest1\DisableAppdomainTest1.csproj", "{CC7072C5-4CD0-4D33-97D0-29DF765AFFC2}"
```

Comment:
Add TestAssets projects to https://github.com/Microsoft/vstest/tree/master/test/TestAssets/TestAssets.sln

Reply:
done